docusaurus/docs/learn/quickstarts/network/local-kubernetes.md

```diff
@@ -475,56 +475,56 @@ Configure CoreDNS in the miniziti cluster. This is necessary no matter which hos
 1. Get the admin password on your clipboard.
 
    ```bash
-   kubectl get secrets "minicontroller-admin-secret" \
+   kubectl get secrets "ziti-controller-admin-secret" \
       --namespace ziti-controller \
-      --output go-template='{{"\nINFO: Your OpenZiti Console http://miniconsole.ziti password for \"admin\" is: "}}{{index .data "admin-password" | base64decode }}{{"\n\n"}}'
+      --output go-template='{{"\nINFO: Your OpenZiti Console http://miniziti-console.ziti password for \"admin\" is: "}}{{index .data "admin-password" | base64decode }}{{"\n\n"}}'
    ```
 
-1. Open [http://miniconsole.ziti](http://miniconsole.ziti) in your web browser and login with username "admin" and the password from your clipboard.
+1. Open [http://miniziti-console.ziti](http://miniziti-console.ziti) in your web browser and login with username "admin" and the password from your clipboard.
 
 ### Create OpenZiti Identities and Services
 
 Here's a BASH script that runs several `ziti` CLI commands to illustrate a minimal set of identities, services, and policies.
 
 ```bash
 ziti edge create identity device "miniziti-client" \
-    --jwt-output-file /tmp/miniziti-client.jwt --role-attributes testapi-clients
+    --jwt-output-file /tmp/miniziti-client.jwt --role-attributes miniziti-httpbin-clients

```

Comment:
I've adapted most of the branding to align with `httpbin`  instead of `miniziti-httpbin`, per your preference. I've only retained `miniziti-httpbin` for the helm release name (it's changing from `miniziti-httpbin-host` to `miniziti-httpbin`) to distinguish it from edge concepts.

As an aside, it would'v been better if the httpbin fork (and the chart) indicated some better naming than the bare `httpbin`, to help distinguish our ziti integrated fork from the parent projects. The `miniziti-httpbin` name here servers also to make that distinction.

If you're good with this compromise, then please consider this resolved.

Reply:
`httpbinz`, perhaps? The name of [the fork](https://github.com/openziti-test-kitchen/go-httpbin) matches the upstream, `go-httpbin`.